### ISSUE TITLE
QA Testing: Add accessibility to emoji picker skin tone selector  

### ISSUE THREAD

**Issue by emilyacook** [mattermost/quality-assurance#13]
QA testing needed for issue https://github.com/mattermost/mattermost-server/issues/18046
# Issue Description
Currently the emoji picker’s default skin tone selector is missing the Aria attributes to be picked up by screen readers. This means we need to add a label, role, description and maybe alter the order on which the tabs goes through so it can be actioned.

while originally the report was concerning JAWS, it requires a license, so having it being read by another screen reader such as the MacOS or Windows default one, would suffice.

Context and Old ticket description:

Environment: daily, Win 10 Desktop app using JAWS reader

Steps:
1) Enable screen reader
2) Open emoji picker, focus on search text field 
3) Use tab to move through items 

Expected: Skin tone selector is read

Actual: Skin tone selector is skipped

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-36733
